torch_xla/csrc/runtime/disc_computation_client.cc

```diff
@@ -135,19 +135,19 @@ std::vector<ComputationClient::DataPtr> DISCComputationClient::TransferToDevice(
 
     auto dtype =
         at::TensorOptions(TorchTypeFromXlaType(tensor->shape().element_type()));
-    auto ret = at::empty(sizes, dtype).contiguous();
+    auto ret = at::empty(sizes, dtype.pinned_memory(true)).contiguous();
     // tensor->populate_fn(tensor, ret.data_ptr(),
     //                    ret.element_size() * ret.numel());
```

Comment:
remove these unused codes?

Reply:
okay, I removed the unused code in PR #20 , that let's merge this PR ASAP.